packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -206,6 +208,17 @@ export default async args => {
                 : null,
             image(),
             production ? terser() : null,
+            config.licenses
+                ? license({
+                      thirdParty: {
+                          includePrivate: true,
+                          output: {
+                              file: join(outDir, "dependencies.txt"),
+                              encoding: "utf-8" // Default is utf-8.
```

Comment:
Do you have a particular reason to leave this comment here? It seems redundant 😄

Reply:
Leftover